### PR TITLE
add functions to test_SimulatedTimeSeries.py

### PR DESCRIPTION
**Pull request by minglong-cse2016**
Updates for test_SimulatedTimeSeries.py:

1. added test_type(), test_str(), test_repr(), test_mean_std to the test module.
2. tested all existing test functions
3. added docstrings to all existing functions
4. Please double check if the test criteria for test_mean_std is good.

### CODE REVIEW

File: timeseries/tests/test_SimulatedTimeSeries.py [hunk 1]
```diff
@@ -1,66 +1,142 @@
-from SimulatedTimeSeries import SimulatedTimeSeries
+""" test code for SimulatedTimeSeries
```
Comment (lyuqin) [inline timeseries/tests/test_SimulatedTimeSeries.py:1]: I am not quite sure whether the doc should be put ahead of imports or not. Double check this format (or use pylint)

File: timeseries/tests/test_SimulatedTimeSeries.py [hunk 2]
```diff
@@ -1,66 +1,142 @@
...
     for value in count():
         if stop and value > stop:
             break
         yield value
 
-def make_stat_data(m, stop=None):
+
+def make_stat_data(in_std, stop=None):
+    """
+    A function to generate normally distributed data with mean of 1e4 and standard deviation m
```
Comment (lyuqin) [inline timeseries/tests/test_SimulatedTimeSeries.py]: "with ... standard deviation m", but it seems that the std here is in_std, not m?

File: timeseries/tests/test_SimulatedTimeSeries.py [hunk 3]
```diff
@@ -1,66 +1,142 @@
...
-
-"""
-TODO:
-Finish the test function for online_mean and online_std when they are ready
-
 def test_mean_std():
-    #test chunk=1 and chunk>1
-    for data_size, chunk_size in [(50, 5), (50,4), (50, 3), (50, 2), (50, 1)]:
+    """
+    A function to test online_mean and online_std using data size 1e5 and chunk size 9e4
```
Comment (lyuqin) [inline timeseries/tests/test_SimulatedTimeSeries.py:104]: chunk size seems way tooooooo big, why not use chunk size < 10?

Comment (lyuqin) [reply timeseries/tests/test_SimulatedTimeSeries.py:104]: I suggest data size = 100, chunk size = 5

**Review by lyuqin — CHANGES_REQUESTED** [review]

File: timeseries/tests/test_SimulatedTimeSeries.py (hunk 2 above)
Comment (minglong-cse2016) [reply timeseries/tests/test_SimulatedTimeSeries.py]: Will correct the variable name.

File: timeseries/tests/test_SimulatedTimeSeries.py (hunk 1 above)
Comment (minglong-cse2016) [reply timeseries/tests/test_SimulatedTimeSeries.py:1]: Pylint seems to suggest this format, thanks!

**Review by lyuqin — APPROVED** [review]

**Pull request merged**